README.md

```diff
@@ -102,8 +102,8 @@ The `"ServiceHandlerSpan"` is _active_ while it's running FunctionA and Function
 
 **The `ScopeManager` API makes it possible to fetch the `span()` in `FunctionA` and re-activate it in `FunctionB`.** Note that every `Tracer` contains a `ScopeManager`. These are the steps:
 
-1. Start a `Span` via either `startManual` or `startActive(false)` to prevent the `Span` from being finished upon `Scope` deactivation.
-2. In the closure/`Runnable`/`Future`/etc itself, invoke `tracer.scopeManager().activate(span, false)` to re-activate the `Span` and get a new `Scope`, then `deactivate()` it when the `Span` is no longer active (or use try-with-resources for less typing).
+1. Start a `Span` via either `start()` or `startActive(false)` to prevent the `Span` from being finished upon `Scope` deactivation.
+2. Pass the span along to the next thread.
```

Comment:
So passing a span to another thread can be done whether the span is active or inactive, but you cannot do things with the span until you call "activate" -- that does the work of saying "start the span (if it is not finished) and set it as the ActiveSpan in this thread"?